pkg/virtctl/create/vm/vm.go

```diff
@@ -457,7 +474,16 @@ func (c *createVM) usage() string {
   {{ProgramName}} create vm --user cloud-user --password-file=/path/to/file
 	
   # Create a manifest for a VirtualMachine with a Containerdisk and a Sysprep volume (source ConfigMap needs to exist)
-  {{ProgramName}} create vm --memory=1Gi --volume-containerdisk=src:my.registry/my-image:my-tag --sysprep=src:my-cm`
+  {{ProgramName}} create vm --memory=1Gi --volume-containerdisk=src:my.registry/my-image:my-tag --sysprep=src:my-cm
+
+  # Create a manifest for a VirtualMachine with SSH public keys injected into the VM from a secret called my-keys to the user also specified in the cloud-init config
+  {{ProgramName}} create vm --user cloud-user --access-cred=type:ssh,src:my-keys
+
+  # Create a manifest for a VirtualMachine with SSH public keys injected into the VM from a secret called my-keys to a user specified as param
+  {{ProgramName}} create vm --access-cred=type:ssh,src:my-keys,user:myuser
+
+  # Create a manifest for a VirtualMachine with password injected into the VM from a secret called my-pws
+  {{ProgramName}} create vm --access-cred=type:password,src:my-pws`
```

Comment:
nit - move these examples above `--sysprep` alongside `--ssh-key` and `password-file`?

Reply:
ACK